### DISCUSSION TITLE
How to remove a TOTP auth factor?

### DISCUSSION THREAD

**Discussion by ashep** [zitadel/zitadel#8052] [Q&A] [answered]
Hi there!

I see two methods in v2beta API:

- [Start the registration of a TOTP generator for a user](https://zitadel.com/docs/apis/resources/user_service/user-service-register-totp)
- [Verify a TOTP generator for a user](https://zitadel.com/docs/apis/resources/user_service/user-service-verify-totp-registration)

Please point me to an API method documentation that allows the removal of the TOTP setup from a user.

Thank you.

**Comment by hifabienne** [comment, marked as answer]
at the moment the remove totp method is not yet implemented in the new beta api.
You can use the current v1 api: https://zitadel.com/docs/apis/resources/mgmt/management-service-remove-human-auth-factor-otp

**Reply by ashep** [reply]
Thank you.
